apps/meteor/tests/end-to-end/apps/video-conferences.ts

```diff
@@ -369,8 +581,13 @@ describe('Apps - Video Conferences', () => {
 						expect(call1).to.have.a.property('createdBy').that.is.an('object');
 						expect(call1.createdBy).to.have.a.property('_id').equal(credentials['X-User-Id']);
 						expect(call1.createdBy).to.have.a.property('username').equal(adminUsername);
+						expect(call1).to.not.have.a.property('discussionRid');
 
 						expect(call2).to.have.a.property('_id').equal(callId2);
+						expect(call2).to.not.have.a.property('discussionRid');
+
+						expect(call3).to.have.a.property('_id').equal(callId3);
+						expect(call3).to.have.a.property('discussionRid').that.is.a('string');
```

Comment:
Is this the expected behavior? The list endpoint returns either type and not consider the setting for the persistent chat, right? Curious.

Reply:
Yes, the setting is considered in the call creation, the list endpoint will return whatever data is there regardless of the setting value.